hifis-net/RSD-as-a-service
# 51 replace halvar

# Replaces Helmholtz Halvar

Fixes #51

Changes proposed in this pull request:

* replaces Helmholtz Halvar with D-DIN
* D-DIN is licensed OFL-1.1-RFN, which allows redistribution, also in modified form, provided that
  > 3) No Modified Version of the Font Software may use the Reserved Font Name(s) unless explicit written permission is granted by the corresponding Copyright Holder. This restriction only applies to the primary font name as presented to the users.
* The font is available as OTF only, so I had to convert it to woff. Since this is a modified version of the original files, I had to rename the font. The font name now is **hifis-din**

How to test:

* `docker-compose up --scale frontend-dev=1 --scale scrapers=0`
* Open http://localhost:3000 and confirm usage of the font

PR Checklist:
